core/src/main/java/org/jboss/jbossset/bugclerk/Candidate.java

```diff
@@ -81,6 +81,15 @@ public void addViolation(Violation violation) {
         this.violations.add(violation);
     }
 
+    public boolean isJIRA() {
+        //return this.bug instanceof JiraIssue;
+        if(bug.getTrackerType().toString().equalsIgnoreCase("jira")) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
```

Comment:
You can do that as part of the check (look at other check which are specific to JIRA). I would rather not add such method if there is an "ok enough" MVEL way of doing that. Neither would I like to add "logic" of anykind to Candidate which aims at being a simple POJO